packages/graphql/src/classes/Executor.ts

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
question: Do we have to make any updates/additions to the documentation?

Reply:
We do not - these changes were documented in #953, and this is just fixing some of the underlying implementation. 🙂